Python/Product/PythonTools/PythonTools/Intellisense/FuzzyCompletionSet.cs

```diff
@@ -221,6 +223,8 @@ public class FuzzyCompletionSet : CompletionSet {
                 }
                 _filteredCompletions.Filter(IsVisible);
             }
+
+            CommitByDefault = DefaultCommitByDefault;

```

Comment:
I like the xml documentation which is easier to find.  With the documentation you could call out its behavior as well.  The public const requires they guess its usage unless they have the code open, in which case its no better than true.  I doubt folks will be fishing through the class via iSense or object explorer when they could just crack code.  Perhaps I'm just missing the programmatic experience you're thinking of.